topics/icrc_31_list_identities.md

```diff
@@ -0,0 +1,67 @@
+# ICRC-31: List Identities (ICRC-25 Extension)
+
+[![Status Badge](https://img.shields.io/badge/STATUS-DRAFT-ffcc00.svg)](https://github.com/orgs/dfinity/projects/31)
+
+
+<!-- TOC -->
+
+- [ICRC-31: List Idenities](#icrc-31-list-identities)
+    - [Summary](#summary)
+    - [Method](#method)
+    - [Request](#request-2)
+    - [Response](#response-2)
+    - [Errors](#errors-2)
+    - [Message Processing](#message-processing-2)
+    - [Example](#example-2)
+
+## Summary
+
+The purpose of the `icrc31_list_identities` message is for the relying party to receive information about the identities managed by the signer.
+
+## Method
+
+**Name and Scope:** `icrc31_list_identities`
+
+**Prerequisite:** Active session with granted permission for method name/scope.
```

Comment:
```suggestion
**Prerequisite:** Active session with granted permission scope `icrc31_list_identities` or `*`.
```

Reply:
Nit: I would make this as clear as possible to the reader what the exact permission needs to be.